app/scripts/views/force_auth.js

```diff
@@ -89,8 +115,12 @@ define(function (require, exports, module) {
       var email = self.relier.get('email');
 
       return self.resetPassword(email)
-        .fail(function (err) {
-          self.displayError(err);
+        .fail(function (error) {
+          if (AuthErrors.is(error, 'UNKNOWN_ACCOUNT')) {
```

Comment:
Reset password behavior update from @ryanfeeley - #3477. "reset password" should go to the "password reset" screen, where the user sees the big warning. No more reset password now. Revert "reset password" behavior here.